internal/service/orginvitation/resource_org_invitation.go

```diff
@@ -50,10 +50,9 @@ func Resource() *schema.Resource {
 				Computed: true,
 			},
 			"teams_ids": {
-				Type:       schema.TypeSet,
-				Optional:   true,
-				Computed:   true,
-				ConfigMode: schema.SchemaConfigModeAttr,
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
```

Comment:
as not only cluster is being changed you might want to change the PR title

Reply:
right! done